README.md

```diff
@@ -1174,12 +1174,15 @@ Usually, instead of restricted stock, an employee will get stock options.
 allow individuals to buy a specified number of shares in the company they work for at a
 fixed price. Stock options are the most common way early-stage companies grant equity.
 
+🄳 A person who has received a stock option grant is not a shareholder until they
```

Comment:
(This is using a term just before it's defined. Tracking this as a consideration separately.)